pymc/distributions/censored.py

```diff
@@ -101,11 +101,20 @@ class Censored(Distribution):
 
     Examples
     --------
+    Censoring with upper & lower points set to +/-1
     .. code-block:: python
 
         with pm.Model():
             normal_dist = pm.Normal.dist(mu=0.0, sigma=1.0)
             censored_normal = pm.Censored("censored_normal", normal_dist, lower=-1, upper=1)
+
+    Partial censoring of normal distributions achienved by passing +/-inf censor points.
+    Examples of 4 censor conditions: uncensored (-inf, inf), upper censored (-inf, 1),
```

Comment:
First emphasize that censoring bounds can change across dimensions of the variable, and then enumerate the examples. If they don't change, one can use `None` as upper or lower, not only +/-np.inf